cart.py

```diff
@@ -1,6 +1,55 @@
+"""Product and Cart class realisation"""
+import csv
+
+
 class Product:
-    pass
+    """Product class"""
+    def __init__(self, name, price, qty):
+        self.name = name
+        self.price = float(price)
```

Comment:
price and qty are numeric values. This class shouldn't know that we've got them from a file.